src/View/SerializedView.php

```diff
@@ -84,8 +85,23 @@ public function render($view = null, $layout = null)
         }
 
         if ($serialize !== false) {
-            return $this->_serialize($serialize);
-        } elseif ($view !== false && $this->_getViewFileName($view)) {
+            $result = $this->_serialize($serialize);
+            if ($result === false) {
+                if (Configure::read('debug') && in_array('trace', $serialize)) {
```

Comment:
Well, it happened in debug mode without a good reason (There was nothing in the stack trace that should have let this happen).
So as a result you would not be able to work locally anymore :/